### PR TITLE
Responsive

### PR DESCRIPTION
**Pull request by jkmaswell**
## Summary

Adding responsiveness to project.

## Screenshots

_users_juancamiloisaza_documents_ji-angular_index html

## Trello Card

https://trello.com/c/pI1K6pMW/15-dise%C3%B1o-responsive

### CODE REVIEW

File: scss/footer.scss [hunk 1]
```diff
@@ -19,4 +19,20 @@
...
+    }
+
+    .footer-logo {
+      margin-bottom: 40px;
+      max-height: 25px;
+    }
+
+    .footer-copyright {
+      align-self: center;
+      font-size: 9px;
```
Comment (FrankIglesias) [inline scss/footer.scss]: use 10px instead

File: scss/styles.scss [hunk 2]
```diff
@@ -20,5 +20,14 @@
     grid-template-columns: repeat(3, 1fr);
     grid-template-rows: auto;
     margin: 0 160px 80px;
+
+    @media screen and (max-width: 1024px) {
```
Comment (FrankIglesias) [inline scss/styles.scss]: define a constant for this media query

File: scss/tech.scss [hunk 3]
```diff
@@ -28,4 +28,19 @@
     grid-area: feature-card;
     justify-content: center;
   }
+
+  @media screen and (max-width: 1024px) {
```
Comment (FrankIglesias) [inline scss/tech.scss]: this media query doesn't have a class associated, where are you applying this styles?

File: scss/tech.scss [hunk 4]
```diff
@@ -28,4 +28,19 @@
...
+    border-bottom: 2px solid $green;
+    border-radius: 0;
+    margin: 0 30px;
+    width: calc(100% - 60px);
```
Comment (FrankIglesias) [inline scss/tech.scss]: aviod this

File: scss/tech.scss (hunk 3 above)
Comment (jkmaswell) [reply scss/tech.scss]: yes, it does, the associated class is .tech

File: scss/styles.scss [hunk 5]
```diff
@@ -1,3 +1,5 @@
+$break-point: 'screen and (max-width: 1024px)';
```
Comment (FrankIglesias) [inline scss/styles.scss]: give a name according to the device

File: scss/styles.scss [hunk 6]
```diff
@@ -1,4 +1,4 @@
-$break-point: 'screen and (max-width: 1024px)';
+$mobile-devices: 'screen and (max-width: 1024px)';
```
[the remark this answers is not in the document]
Comment (FrankIglesias) [reply scss/styles.scss]: however as a first approach it's ok

[the remark this answers is not in the document]
Comment (FrankIglesias) [reply scss/styles.scss]: i'd also separate this into another file like `resolutions.scss`

**Pull request merged**